src/python/qeqiskit/simulator/simulator.py

```diff
@@ -262,8 +258,8 @@ def get_wavefunction(self, circuit):
         Returns:
             pyquil.wavefunction.Wavefunction
         """
-        self.num_circuits_run += 1
-        self.num_jobs_run += 1
+        self.number_of_circuits_run += 1
+        self.number_of_jobs_run += 1
```

Comment:
Here, I think we should call `super().get_wavefunction(circuit)`

Reply:
Addressed in [d81040](https://github.com/zapatacomputing/qe-qiskit/pull/18/commits/d8104090971fb60f46de371cafe4e6e745c211d3).